### PR TITLE
Daily Coding Practice 8 is added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/FilterString_ReadMe [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"C:\Program Files\Java\jdk-17.0.4.1\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\lib\idea_rt.jar=61432:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\suyogw\Desktop\Auto\NewMyWayRepo\simple\out\production\simple codingPractice.FilterString
+The Number in String is : 0
+The Alphabet  in String is : uyg
+The special symbol in String is : $
+
```
Comment (Ams12) [inline simple/src/codingPractice/FilterString_ReadMe:5]: Remove extra space

File: simple/src/codingPractice/MixString.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    public void separateSrting(String mixName)
+    {
+        StringBuffer alphaCharacter = new StringBuffer();
+        StringBuffer numberCharacter = new StringBuffer();
+        StringBuffer specialCharacter = new StringBuffer();
+
+        for (int index =0; index<mixName.length();index++)
+        {
+            if(Character.isDigit(mixName.charAt(index))){
+
```
Comment (Ams12) [inline simple/src/codingPractice/MixString.java:13]: Remove extra space

File: simple/src/codingPractice/FilterString_ReadMe (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/FilterString_ReadMe:5]: Done

File: simple/src/codingPractice/MixString.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/MixString.java:13]: Done

**Review by Ams12 — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**